PyMca5/PyMcaGui/io/QTiledWidget.py

```diff
@@ -32,6 +34,8 @@ def __init__(self, model=None, dialog_model=None):
         super().__init__()
         if dialog_model is None:
             dialog_model = TiledCatalogSelector()
+            # dialog_model = TiledCatalogSelector("https://tiled.nsls2.bnl.gov/api")
+            # dialog_model = TiledCatalogSelector("https://tiled-demo.blueskyproject.io/api")
```

Comment:
These should probably move to an environment variable or other configuration.  For now stash these in your local repo, rather than bring them into the project code.
```suggestion
```